MailerQ.Test/ApiClientTest.cs

```diff
@@ -0,0 +1,208 @@
+﻿using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Moq;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Text;
+using Xunit;
+
+namespace MailerQ.ApiClient.Test
+{
+    public class ApiClientTest
+    {
+        private const string _url = "http://192.168.170.140";
+        private const string _token = "sBftYte4RhcXe4BHL0PccgyGGdOrrYD13gOinbhNCeI6bLwv6nQz7BM1pfl8CdrnAPXZpMJnJxTvtDB4j4BW1xd3V1SksNFhA4xvPBOnS1IasoyeLqc3z6dcdC4PNqsGMRIPyTWZOqRYTJcFfvPnueGsVoq5bDcEujmY6lgD4tOKb4fe8c88JCM2vAr6P4cLmjJAA76IOkBKyj4ZwuFP2M2PTmbmEFayDoX2iQNKbeC4IiJQT6MDt3VAo8bM448";
```

Comment:
Oh, I did not saw it. It is a public repository, is this token a secret?

In that case, removing from here is not enough since _nothing is deleted from thne Internet_, we would need to revoke the token.

Reply:
I didn't think about the public repo. :( Now, that token was removed, it is not valid anymore. Also, I will update Test project in order to configure token value in other way.